### ISSUE TITLE
Add Contributer/Developer guide/help page to website

### ISSUE THREAD

**Issue by fschueler** [stratosphere/peel#65]
I think it would be helpful to add a contributers/developers page to the docs on the website.
This should explain how to setup a development environment for Peel. how to add Systems etc. and how to setup debugging for example in Intellij.

**Comment by aalexandrov** [comment]
:+1: from my side. 

I actually wanted to structure this as a blog post with with a running example based on the recent `DStat` addition from @verbit, but I am open to other systems if we find a volunteer to do this.

**Comment by joh-mue** [comment]
+1

I am trying to extend peel to get multiple jobs to run on Yarn concurrently to simulate cluster workloads. For this purpose I would like to be able to debug what I modified in peel.

**Comment by aalexandrov** [comment]
I think that @akunft had something written up already in [the Google doc](https://docs.google.com/document/d/1ZDbDFN3uZsVGLz8H7bwowyfLCZ8xlzoE3-aOVN5HGB4/edit#) for the Manual but I cannot find it.

The basic method that I use is to configure a new "Run Configuration" in IntelliJ that uses `$BUNDLE_BIN/bundlename` as a current directory while executing.

**Comment by aalexandrov** [comment]
@joh-mue here is a screenshot of an example "Run configuration" on my machine that starts Yarn. I have similar configurations for each system that I want to test. 

If you wish to develop a special Experiment bean I suggest to use `exp:run --just` and start / stop the associated systems with a dedicated `exp:setup` and `exp:teardown` calls.

![screenshot from 2015-11-20 14 54 14](https://cloud.githubusercontent.com/assets/1071946/11301351/b47073aa-8f96-11e5-93a5-897739566a61.png)

**Comment by akunft** [comment]
@aalexandrov  yes I did, but we wanted to move it to another section and it somehow got lost... Is there a option to see older versions of an document in google docs?

**Comment by aalexandrov** [comment]
I think so. You can look [at the revision history](https://docs.google.com/document/d/1ZDbDFN3uZsVGLz8H7bwowyfLCZ8xlzoE3-aOVN5HGB4/edit#heading=h.culgy8pg5r5u).

**Comment by akunft** [comment]
Nvm, added the section about *Setup/run in IDE* to the document again under *Development*

**Comment by joh-mue** [comment]
Thanks for the help. I tried the above but still get this weird scalac error which I pasted into this gist: https://gist.github.com/joh-mue/d098fcbe35b39d343789

Probably something related to my setup though. I'll try reinstalling everything and see if that helps.

**Comment by aalexandrov** [comment]
This seems to be a problem with the compilation, not with the run configuration.

**Comment by trenner** [comment]
What version of scala plugin for intellij  are you using? 
I had some similar problems, but after updating to  intelij 15 + scala plugin 2.0.4 it works.

**Comment by joh-mue** [comment]
I just updated too, rebuild the entire local maven repository, ran ```mvn clean``` and donated $5 to apache. Not sure which it was but one of those measures fixed the issue. It's working for me now. Thanks for the advice.

**Comment by aalexandrov** [comment]
My bet is on `mvn clean`, although the `$5` seems plausible as well.

**Comment by zerg-junior** [comment]
Are there any other developer guides besides the draft  on Google Docs ?

**Comment by aalexandrov** [comment]
Not really, but contributions are welcome. At the moment the best guide is to look at existing implementations. To work out the details, it helps if you are familiar with the [Peel Execution Lifecycle](http://peel-framework.org/manual/execution-workflow.html#execution-lifecycle) implemented by the [`suite:run` command](https://github.com/stratosphere/peel/blob/master/peel-core/src/main/scala/org/peelframework/core/cli/command/suite/Run.scala).
